### PR TITLE
mixtral: small cleanups and ability to supply initial tokens

### PR DESCRIPTION
**Pull request by tmc**
Small cleanups of the mixtral example.

### CODE REVIEW

File: examples/mixtral.py [hunk 1]
```diff
@@ -44,13 +45,12 @@ def __call__(self, x:Tensor) -> Tensor:
...
   spp = SentencePieceProcessor(model_file=args.weights + "/tokenizer.model")
-
-  toks = [spp.bos_id()]
+  toks = [spp.bos_id()] + spp.encode(args.initial_tokens)
   start_pos = 0
   for i in range(args.count):
     GlobalCounters.reset()
     with Timing("total ", enabled=args.timing, on_exit=lambda x: f", {1e9/x:.2f} tok/sec"):
-      tok = model(Tensor([toks[start_pos:]]), 0 if start_pos == 0 else Variable("start_pos", 1, 1024).bind(start_pos), args.temperature).multinomial().item()
+      tok = model(Tensor([toks[start_pos:]]), start_pos, args.temperature).multinomial().item()
```
Comment (tmc) [inline examples/mixtral.py:53]: having Variable in here was/is blowing up in https://github.com/tinygrad/tinygrad/blob/59ab3675a3cfedcd0d443d6290653ff8c16b8d77/tinygrad/tensor.py#L651

**Comment by geohot** [comment]
Did you test this?

**Comment by geohot** [comment]
Oh, I guess it will run....but this doesn't work. The wrong experts are chosen for the tokens after the first on the first run

**Comment by tmc** [comment]
closing

**Pull request closed without merging**